### PR TITLE
Add script for OpenVINO installation on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by H-HChen**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -30,6 +30,18 @@ else
     echo "Skip installing ROS"
 fi
 
+if [ "$ros2_distro" '==' "foxy" ]; then
```
Comment (evshary) [inline install.sh]: You also need to check whether the architecture is x64 or not.

File: install.sh [hunk 2]
```diff
@@ -30,6 +30,18 @@ else
...
+    # OpenVINO environment installation
+    echo -n "Do you want to install OpenVINO automatically? (y/N): "
```
Comment (evshary) [inline install.sh]: "Do you want to install Intel RealSense SDK and OpenVINO automatically?"
"Note that if you choose yes, it means you agree to the Intel software license."
"Install or not? (y/N): "

**Review by evshary — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (H-HChen) [reply install.sh]: The output message has been modified to requested content as image below.
Screenshot from 2021-08-17 19-24-09

File: install.sh (hunk 1 above)
Comment (H-HChen) [reply install.sh]: Now there is a check process before OpenVINO installation to ensure that CPU vendor is Intel.

**Comment by evshary** [comment]
Due to summer school, I prefer not to merge into master now.
I'll create openvino branch temporarily.

**Pull request closed without merging**